### PR TITLE
Initial station list endpoints and examples

### PR DESCRIPTION
**Pull request by huksley**
Adds station information API
See deployed documation here:
https://huksley.github.io/maas-tsp-api/redoc.html

TODO: define linking between stations and booking options offered

### CODE REVIEW

File: specs/booking.yml [hunk 1]
```diff
@@ -721,6 +726,110 @@ paths:
...
+      responses:
+        '200':
+          description: The stations matching the query
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/tsp/stations-list/response.json
+              examples:
+                Bikes1:
+                  summary: Docker bike stations
```
Comment (brylie) [inline specs/booking.yml]: Should this be "Docked"?

File: specs/booking.yml [hunk 2]
```diff
@@ -721,6 +726,110 @@ paths:
...
+            $ref: ../schemas/core/components/station.json#/definitions/id
+        - name: agencyId
+          description: TSP provider id which this station belongs to
+          in: query
+          required: true
+          schema:
+            $ref: ../schemas/core/components/common.json#/definitions/agencyId
+      responses:
+        '200':
+          description: Information about single station
```
Comment (brylie) [inline specs/booking.yml]: "Information about **a** single station" :-)

File: specs/booking.yml [hunk 3]
```diff
@@ -721,6 +726,110 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/tsp/stations-retrieve/response.json
+              examples:
+                Bikes1:
+                  summary: Docker bike stations
```
Comment (brylie) [inline specs/booking.yml]: "Docked bike stations"?

**Comment by brylie** [comment]
This PR is looking good. I added just a couple of grammar remarks :-)

File: specs/booking.yml (hunk 2 above)
Comment (huksley) [reply specs/booking.yml]: 👍

File: specs/booking.yml (hunk 1 above)
Comment (huksley) [reply specs/booking.yml]: Too much docker for me)) fixed

**Comment by huksley** [comment]
Thanks, fixed!

**Pull request merged**